Index: templates/input.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>{% extends 'base.html' %}\n\n{% set active_page = \"input\" %}\n\n{% block content %}\n<div class=\"container\">\n    <div class=\"center-content\">\n    <p class=\"welcome-text\">Type in your URLs here</p>\n    \n    <form class=\"search-form\">\n    <div class=\"input-group\">\n    <textarea id=\"urls\" name=\"urls\" placeholder=\"Enter URLs separated by commas\" required></textarea>\n    <button type=\"submit\">Scrap</button>\n    </div>\n    </form>\n    \n    <div id=\"results\">\n    <!-- Display the results or relevant information here -->\n    </div>\n    </div>\n    </div>\n{% endblock %}\n\n{% block title %}\n    Input Webpage URLs\n{% endblock %}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/templates/input.html b/templates/input.html
--- a/templates/input.html	(revision 5099086e06b5c46c9ca425ec8f3a36e7209b4916)
+++ b/templates/input.html	(date 1700559529918)
@@ -3,6 +3,7 @@
 {% set active_page = "input" %}
 
 {% block content %}
+
 <div class="container">
     <div class="center-content">
     <p class="welcome-text">Type in your URLs here</p>
